pkg/operator/targetconfigcontroller/targetconfigcontroller_test.go

```diff
@@ -605,3 +616,524 @@ func (l *configMapLister) ConfigMaps(namespace string) corev1listers.ConfigMapNa
 func (l *configMapLister) Get(name string) (*corev1.ConfigMap, error) {
 	return l.client.CoreV1().ConfigMaps(l.namespace).Get(context.Background(), name, metav1.GetOptions{})
 }
+
+func TestManageClientCABundle(t *testing.T) {
+	cert1, err := generateTemporaryCertificate()
+	require.NoError(t, err)
+
+	cert2, err := generateTemporaryCertificate()
+	require.NoError(t, err)
+
+	tests := []struct {
+		name                string
+		existingConfigMaps  []*corev1.ConfigMap
+		expectedChanged     bool
+		expectedAnnotations map[string]string
```

Comment:
would it be hard to slightly change these unit tests to include the expectedConfigMap here? It would be easier to review (compare existing with expected) rather than reading the content of the t.Run function.

Reply:
Good idea, done